jcapi-tags.go

```diff
@@ -145,23 +130,33 @@ func (jc JCAPI) GetTagByName(tagName string) (tag JCTag, err JCError) {
 // Add or Update a tag in place on JumpCloud
 //
 func (jc JCAPI) AddUpdateTag(op JCOp, tag JCTag) (tagId string, err JCError) {
-	data, err := tag.MarshalJSON()
+	data, err := json.Marshal(tag)
```

Comment:
Just a note so its captured for others.  If your struct implements `MarshalJSON`, `json.Marshal` should actually call that itself, no need to ever directly call it.